### DISCUSSION TITLE
Monitoring von Auswahl

### DISCUSSION THREAD

**Discussion by Graefer** [brutella/hkknx-public#342] [Ideas]
Mit dem Monitor läßt sich per Tippen die Temperatur eines einzelnen Sensors anzeigen. Es wäre schön, wenn man nun per weiterem Tip Sensoren **dazu** selektieren könnte (anstatt den Sensor zu wechseln). Die Auswahl der Datenquellen über das Dreipunkt-Menü sehe ich eher als Voreinstellung an.

**Comment by brutella** [comment]
Wenn man bei der Auswahl einer anderen Datenquellen entweder die Shift (`⇧`) oder Command (`⌘`)-Taste gedrückt hält, kann man mehrere Datenquellen aus- und abwählen. Funktioniert halt nur bei angeschlossener Tastatur.
